README.md

```diff
@@ -420,11 +421,105 @@ description: "This high priority class should be used for MarkLogic pods only."
+
+This section describes three upgrade procedures:
+1. [Upgrading MarkLogic Kubernetes Helm Chart version](#upgrading-markLogic-version-in-your-release)
+2. [Upgrading MarkLogic version in your release](#upgrading-markLogic-version-in-your-release)
+3. [Upgrading MarkLogic version and MarkLogic Kubernetes Helm Chart version at once](#upgrading-markLogic-version-and-markLogic-kubernetes-helm-chart-version-at-once)
+
+## Upgrading MarkLogic Kubernetes Helm Chart Version
+
+This upgrade is required whenever there is a requirement at your end or when MarkLogic Kubernetes Helm Chart releases a new version with new features or bug fixes. The general procedure to upgrade MarkLogic Kubernetes Helm Chart is as follows:
+
+1. Update the chart repository using the below command to get the new version of the chart
+```
+helm repo update
+```
+2. Check the available upgrades of MarkLogic Kuberenetes Helm chart using following command,
+```
+helm search repo marklogic/marklogic
+```
+3. Make sure the upgrade strategy in values.yaml file is set to OnDelete. 
+```
+updateStrategy: 
+    type: OnDelete
+```
+4. Update the values.yaml file with new values from the updated chart version.
+5. Run helm upgrade command and specify the name of your release, new chart version using --version option and values.yaml file using `-f` option.
+```
+helm upgrade <your-release> marklogic/marklogic -f values.yaml --version <new version>
+```
+6. Terminate the pod with smallest ordinal that is running bootstrap node to start the upgrade. 
+```
+kubectl delete pod <pod-name>
+e.g: kubectl delete pod dnode-group-marklogic-0
+```
+
+Once the pod is terminated, a new pod will be created with updated helm chart version. 
+7. Repeat termination for all pods in your release and complete the upgrade process.
+
+## Upgrading MarkLogic Version in your release
+
+MarkLogic release is independent of MarkLogic Kubernetes Helm Chart release. This upgrade may be required when there is a new MarkLogic version available. Note MarkLogic is also released in Major, Minor or Patch form, pleasr refer [MarkLogic Upgrade Support](https://docs.marklogic.com/11.0/guide/release-notes/en/installation-and-upgrade.html#UUID-ae8580bb-c23d-dda7-593c-0610a633c771_section-idm203341859110072)
+
+Following is the procedure to upgrade MarkLogic version in your release:
+
+1. Update the image.repository and image.tag in the values.yaml file to new version of MarkLogic you want to upgrade to.
+```
+image:
+  repository: marklogicdb/marklogic-db
+  tag: <new tag>
+```
+2. Make sure the upgrade strategy in values.yaml file is set to OnDelete. 
+```
+updateStrategy: 
+    type: OnDelete
+```
+3. Upgrade the helm chart using helm upgrade command with release name, chart name and values.yaml.
+```
+helm upgrade <release-name> <chart-name> -f <values.yaml> 
```

Comment:
We should specify versions here, or it will accidentally upgrade to the latest version